aten/src/ATen/FunctionalTensorWrapper.cpp

```diff
@@ -521,5 +585,82 @@ void setFunctionalizationReapplyViewsTLS(bool reapply_views) {
 }
 
 } // namespace impl
+
+
+void functionalize_op_helper(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
```

Comment:
would be nice to know at a high level what this function does

Reply:
There's a light description in the header, but I can describe in a bit more detail